### DISCUSSION TITLE
The application will stop work at block...

### DISCUSSION THREAD

**Discussion by chiaXXL** [Chia-Network/chia-blockchain#2343] [General]
Today I see the follow message on all of my chia wallets:
[https://paste.pics/79d911a89c803e5bb08432a4cf0de3d6](url)

**Comment by cccat6** [comment]
Same, two of my machine show the same thing.
Any idea?
![image](https://user-images.githubusercontent.com/22387156/115667917-fbc00300-a378-11eb-9931-b3537d1059d7.png)

**Comment by cyr616** [comment]
upgrade to 1.1.0 can slove. but you can‘t delete your polt in this version.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
